gui_qt/project_analysis_workflow.py

```
@@ -42,14 +49,16 @@ def generation_facts_from_output(output: str) -> list[str]:
     return facts
+
+
 STEP_TEXT = {
     "project-analysis-ingest-keywords": (
         "正在导入剧情概要",
         "正在把已确认的关键词剧情概要导入项目分析存储。",
     ),
     "project-analysis-build-structure": (
         "正在构建项目结构",
-        "正在静态解析 label、jump 与 route；不会修改游戏脚本。",
+        "正在读取剧情节点、跳转与路线；不会修改游戏脚本。",
```

Ruff (0.15.21)

[warning] String contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF001)